package.json

```diff
@@ -152,6 +152,7 @@
     "jsonlint-mod": "^1.7.4",
     "lighthouse-logger": "^1.2.0",
     "lodash.isequal": "^4.5.0",
+    "lodash.set": "^4.3.2",
```

Comment:
bundlephobia says this is 2KB minified+gzipped, so basically a nonfactor for folks already invoking `npm install lighthouse` 😆